helpers/isStaticExportProject.js

```diff
@@ -4,9 +4,14 @@
 
 const isStaticExportProject = ({ build, scripts }) => {
   const NEXT_EXPORT_COMMAND = 'next export'
-  const isSetInNetlifyConfig = build && build.command && build.command.includes(NEXT_EXPORT_COMMAND)
+
+  if (!build || !build.command) return
```

Comment:
`build` is always defined, so we might be able to only check `if (!build command)`.

Reply:
cool, will add. just wondering, is there any harm in checking for something even if you expect it to always exist?

Reply:
It does not do any harm, except to code simplicity (adding some code that is unused).